### PR TITLE
add Intro model

### PR DESCRIPTION
**Pull request by hofanguyenn**
Please watch this pull request.

### CODE REVIEW

File: intro_git/models.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
 # Create your models here.
+
+class Intro(models.Model):
+    username = models.CharField(max_length=120)
+    link_git = models.CharField(max_length=256)
+    content = models.TextField()
```
Comment (KickSone) [inline intro_git/models.py:8]: add attribute to link_git.

**Review by KickSone — COMMENTED** [review]
Let check it

File: intro_git/models.py (hunk 1 above)
Comment (hofanguyenn) [reply intro_git/models.py:8]: Okey, I changed it.

**Review by KickSone — APPROVED** [review]
okey!

**Pull request merged**